### PR TITLE
Add a go-releaser config.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This is an example .goreleaser.yml file with some sane defaults.
```
Comment (naveensrinivasan) [inline .goreleaser.yml:1]: How can we test this?

File: .goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
```
Comment (naveensrinivasan) [inline .goreleaser.yml:27]: The auto-generated changelog are not encouraged by CII https://bestpractices.coreinfrastructure.org/en/criteria/0#0.release_notes. What if we turn off the release notes generation?

File: .goreleaser.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# This is an example .goreleaser.yml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod download
+    # you may remove this if you don't need go generate
+    - go generate ./...
```
Comment (naveensrinivasan) [inline .goreleaser.yml:8]: Is this required?

**Comment by naveensrinivasan** [comment]
@dlorenc The #194 took care of the goreleaser config.

**Comment by inferno-chromium** [comment]
Closing as not applicable anymore.

**Pull request closed without merging**